### ISSUE TITLE
would it be possible to edit the world id with this?

### ISSUE THREAD

**Issue by lllXlll** [spinclick/twld#1]

**Comment by jnlon** [comment]
Yes, functionality to update world data is planned. This will take some time though, since re-saving the world into a valid .wld file is non-trivial.

I should have more time to work on this after my exams in mid-December.
